tool/gh_codespaces/install_flutter.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright (C) 2023 Intel Corporation
```

Comment:
header of this script is wrong (copyright year, name of file, description, author, date, etc.)